Amplify/Categories/DataStore/Model/Lazy/LazyReference.swift

```diff
@@ -96,11 +96,15 @@ public class LazyReference<ModelType: Model>: Codable, _LazyReferenceValue {
         }
         let json = try JSONValue(from: decoder)
         switch json {
-        case .object:
+        case .object(let _json):
```

Comment:
As per our code convention, we avoid using prefixed `_` in local variable names. Could you please change it to either `json` or `object`?